proof/invariant-abstract/AARCH64/ArchTcbAcc_AI.thy

```diff
@@ -231,6 +233,34 @@ lemma arch_thread_set_vcpus_of[wp]:
   apply (clarsimp simp: obj_at_def opt_map_def)
   done
 
+\<comment> \<open>FIXME: Shouldn't state_hyp_refs_of be generic? If it was, then this and
+          arch_thread_set_sym_refs_hyp' could be generic as well\<close>
```

Comment:
good question. I have no idea, when I did the split it looked like it shouldn't be, or at least its definition shouldn't be. @lsf37 do you remember discussing this by any chance?

Reply:
also `FIXME arch-split` please

Reply:
ok, that sounds promising then